darts/models/forecasting/forecasting_model.py

```diff
@@ -384,18 +640,12 @@ def historical_forecasts(
             Whether to print progress
         Returns
         -------
-        TimeSeries or List[TimeSeries]
-            By default, a single ``TimeSeries`` instance created from the last point of each individual forecast.
-            If `last_points_only` is set to False, a list of the historical forecasts.
+        TimeSeries or List[TimeSeries] (or Sequence[TimeSeries] or Sequence[List[TimeSeries]])
```

Comment:
Actually the returned Sequence will always be a List, right? You could indicate it in the type hints.